lib/apiservers/engine/backends/volume.go

```diff
@@ -37,5 +37,9 @@ func (v *Volume) VolumeCreate(name, driverName string, opts map[string]string) (
 }
 
 func (v *Volume) VolumeRm(name string) error {
-	return fmt.Errorf("%s does not implement volume.VolumeRm", v.ProductName)
+	client, err := PortLayerClient()
+	if err == nil {
+		return fmt.Errorf("Unable to get Portlayer Client")
```

Comment:
You should change this.  See container.go.  Import > derr "github.com/docker/docker/errors" and wrap that error with derr.NewErrorWithStatusCode().